src/main/java/uk/gov/hmcts/reform/sendletter/config/RedisConfiguration.java

```diff
@@ -30,14 +35,14 @@ public SentLettersCache getRedisCache(
 
         return new SentLettersRedisCache(
             Redisson.create(config),
-            new LetterChecksumGenerator(),
+            context.getBean(LetterChecksumGenerator.class),
```

Comment:
would it work by just adding `LetterChecksumGenerator` to this method params?

Reply:
you mean instead of `context.getBean...` or instead of `..LetterChecksumGenerator.class)`? eitherway won't work - need a class signature or an actual class instance accordingly